exercise9.js

```diff
@@ -0,0 +1,11 @@
+function calculateMatrixSum(matrix) {
+  let sum = 0;
+  const rowLength = matrix.length;
+  const columnLength = matrix[0].length;
+
+  for (let row = 0; row < rowLength; row++)
+    for (let column = 0; column < columnLength; column++)
+      sum += matrix[row][column];
```

Comment:
I would suggest using array methods whenever its possible because they're easily understandable.

Reply:
On it!